Launcher/runtime/config.js

```diff
@@ -41,4 +41,12 @@ var serversConfig = {
         }
         return serversConfig[profile][property];
     }
-};
\ No newline at end of file
+};
+
+var optModNames = {
+	optAutoModName: true,//Попытатся автоматически создать представляемое имя модификации
+	modName: {//"Путь до опц. модификации" : "Отображаемое клиенту имя модификации"
+	  "mods/(TEST)sampler-1.80.jar": "Sampler",
```

Comment:
Удалить или закоментировать эти строчки
```suggestion
	  "mods/(TEST)sampler-1.80.jar": "Sampler",
```